@xen-orchestra/backups/writers/_AbstractWriter.js

```diff
@@ -0,0 +1,31 @@
+const { asyncMap } = require('@xen-orchestra/async-map')
+const { createLogger } = require('@xen-orchestra/log')
+
+const { BACKUP_DIR } = require('../_getVmBackupDir')
+
+const { debug } = createLogger('xo:backups:AbstractWriter')
+
+exports.AbstractWriter = class AbstractWriter {
+  constructor({ adapter } = {}) {
+    this._adapter = adapter
+  }
+
+  async cleanCorruptedBackups() {
```

Comment:
Find a more generic name, that could be used by CR and DR writers, think something like a hook.

Reply:
What's the main idea of creating this method? Because, as I understand, it's very specific to remotes.